tests/cpu/Config_tests.cpp

```diff
@@ -212,6 +212,39 @@ OCIO_ADD_TEST(Config, simple_config_with_duplicate)
                           "Key-value pair with key 'name' specified more than once. ");
 }
 
+OCIO_ADD_TEST(Config, simple_config_with_cdltransform_duplicate)
+{
+
+    constexpr char SIMPLE_PROFILE[] =
+        "ocio_profile_version: 2\n"
+        "search_path: luts\n"
+        "roles:\n"
+        "  default: raw\n"
+        "file_rules:\n"
+        "  - !<Rule> {name: Default, colorspace: default}\n"
+        "displays:\n"
+        "  Disp1:\n"
+        "    - !<View> {name: View1, colorspace: raw}\n"
+        "active_displays: []\n"
+        "active_views: []\n"
+        "colorspaces:\n"
+        "  - !<ColorSpace>\n"
+        "    name: raw\n"
+        "    to_reference: !<GroupTransform>\n"
+        "      direction: forward\n"
+        "      children:\n"
+        "        - !<CDLTransform>\n"
```

Comment:
It could be simplified to:
` to_reference: !<CDLTransform> {slope: [1, 2, 1], slope: [1, 2, 1]}\n"`